frontend/ui/widget/opdsbrowser.lua

```diff
@@ -474,6 +474,8 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     -- download to user selected directory or last opened dir
     local lastdir = G_reader_settings:readSetting("lastdir")
     local download_dir = G_reader_settings:readSetting("download_dir") or lastdir
+    item.author = item.author:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
+    item.title = item.title:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
```

Comment:
Agreed, let's create a helper function in utils to handle filesystem filename compatibility issues. KOReader also runs on systems that does these characters in filenames. Smart detection for inferior filesystems at runtime would be ideal. But you can skip it if you don't want to work on it.

Also, can we replace those characters with something like `-` or `_`?